app/views/templates/script/convert2rhel_analyze.erb

```diff
@@ -5,7 +5,20 @@ model: JobTemplate
 job_category: Convert 2 RHEL
 provider_type: script
 kind: job_template
+template_inputs:
+- name: ELS
+  required: true
+  input_type: user
+  description: Use an Extended Lifecycle Support (ELS) add-on subscription
+  advanced: false
+  value_type: plain
+  options: "true\nfalse"
+  default: "true"
+  hidden_value: false
```

Comment:
After applying this patch, this option is not appearing in the inputs section on the Run Job page in UI for this job template, so just curious to know where do we define this template_inputs for UI?